csharp/ql/lib/semmle/code/csharp/dataflow/internal/SsaImpl.qll

```diff
@@ -958,7 +960,7 @@ private module Cached {
   }
 
   cached
```

Comment:
I think it's OK to replace this with `pragma[nomagic]`; I highly doubt anyone actually calls this.